COMPILING.md

```diff
@@ -64,47 +73,56 @@ make
 
 ---
 
 ## Windows
 
-You will need [Qt](https://www.qt.io/download)
+### Install dependencies
+
+Install [Qt](https://www.qt.io/download) e.g. via the official installer.
 
+**Note:**
 - Use the free GPLv2 license for Open Source development
 - Select Components during installation: Expand the Qt section, find the matching version, preferrably **Qt 5.15.2**, and add the compiler components for your compiler, e.g., `MSVC 2019 32-bit/64-bit` for Visual Studio 2019
-- [ASIO development files](https://www.steinberg.net/en/company/developer.html)
 
-### Compiling and building installer
+If you build with *JACK* support, install JACK via choco: `choco install --no-progress -y jack`
+
+If you build with *ASIO* support, you'll need the [ASIO development files](https://www.steinberg.net/en/company/developer.html). Please ensure you read the ASIO-SDK licensing terms and register with Steinberg if necessary.
 
-Most users will probably want to use this method:
+### Compiling and building installer
 
 1. Open PowerShell
 1. Navigate to the `jamulus` directory
 1. To allow unsigned scripts, right-click on the `windows\deploy_windows.ps1` script, select properties and allow the execution of this script. You can also run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. (You can revert this after having run this script. For more information see the [Microsoft PowerShell documentation page](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy)).
-1. Edit the $QtCompile32 and $QtCompile64 variables.
-1. Run the Jamulus compilation and installer script in PowerShell: `.\windows\deploy_windows.ps1`.
+1. Run the Jamulus compilation and installer script in PowerShell: `.\windows\deploy_windows.ps1 "C:\Qt\<pathToQt32BitVersion>" "C:\Qt\<pathToQt64BitVersion>"`.
 1. You can now find the Jamulus installer in the `.\deploy` directory.
 
 ### Compiling only
 
 1. Create a folder under `\windows` called ASIOSDK2
-1. Download the [ASIOSDK](https://www.steinberg.net/asiosdk), open the top level folder in the .zip file and copy the contents into `[\path\to\jamulus\source]\windows\ASIOSDK2` so that, e.g., the folder `[\path\to\jamulus\source]\windows\ASIOSDK2\common` exists
+1. Download the [ASIOSDK](https://www.steinberg.net/asiosdk) if not already done, open the top level folder in the .zip file and copy the contents into `[\path\to\jamulus\source]\windows\ASIOSDK2` so that, e.g., the folder `[\path\to\jamulus\source]\windows\ASIOSDK2\common` exists
 1. Open Jamulus.pro in Qt Creator, configure the project with a default kit, then compile & run
 
 ---
 
 ## macOS
 
-You will need XCode and Qt as follows:
+### Install dependencies
+
+You will need Xcode and Qt.
+
+First, install [Xcode from the Mac AppStore](https://apps.apple.com/us/app/xcode/id497799835?mt=12). Then [install homebrew](https://brew.sh/).
+
+After that you can install Qt via homebrew:
```

Comment:
If homebrew really doesn't work anymore for this (I can't tell), we could guide users to use aqt instead, similar to CI?

Reply:
Probably it installs Qt6 or similar? Maybe worth asking on the homebrew repo

Reply:
i.e. See: https://formulae.brew.sh/formula/qwt-qt5

The homebrew link changes a lot and is very confusing.
It used to be `brew install Qt5` then changed to `brew install Qt@5` it now seems to be `brew install qwt-qt5` but this one currently installs Qt6 !!!
So I think it's better to point to the official Qt pages, though these are very confusing too. It's very hard to find the correct version on the official Qt pages too, so I think we should be as specific as possible hereto prevent downloading the wrong Qt version.  (We could use https://www.qt.io/offline-installers, but there are other, equally hard to find, pages on qt.io too.  ) 
Also don't forget to mention which options to install, this is especially important for Android and iOS builds. (And AFAIK homebrew doesn't give you any of these options at all.)